### DISCUSSION TITLE
Can't Install Directus 9 in Nodejs supported Shared Host

### DISCUSSION THREAD

**Discussion by ghezeli** [directus/directus#5574] [Archive]
Hi
Tanks for this product
I try to install Directus 9 in my resellery but can't run it
Please help me how to do this
My Resellery is :
https://www.netcup.eu/bestellen/produkt.php?produkt=2425

**Comment by hermesalvesbr** [comment]
I make it work on the centova web panel (cwp), what I needed was to have access to the shell(ssh). To execute the project creation, and then on the panel I put the command to run the project (npx directus start). Then talk to your hosting to run the start command. See if it will be necessary to change the project port in .env

**Comment by Dominic-Marcelino** [comment]
Wrote a step by step guide https://discord.com/channels/725371605378924594/741330910594465823/840938171071463454
